extensions/interactions/MultipleChoiceInput/protractor.js

```diff
@@ -59,14 +59,10 @@ var answerObjectType = 'NonnegativeInt';
 var testSuite = [{
   interactionArguments: [[function(editor) {
       editor.appendBoldText('right');
-    }, function(editor) {
```

Comment:
You can't just remove these lines as they are needed for the 'wrong' answer submission below. This is probably why Travis is failing. I suggest you change their formatting to italic instead.